### PR TITLE
test(blade): add eslint plugins for a11y

### PR DESCRIPTION
**Pull request by anuraghazra**
Added
- https://github.com/FormidableLabs/eslint-plugin-react-native-a11y
- https://github.com/jsx-eslint/eslint-plugin-jsx-a11y

### CODE REVIEW

**Comment by kamleshchandnani** [comment]
I'm surprised validate check didn't fail after adding the eslint plugin? 🤔 everything is accessible in it's current state?

**Comment by anuraghazra** [comment]
> everything is accessible in it's current state or are we missing any configuration?

If there is no aria. there's no errors :p 

> There was errors but in blade-old I've ignored those, not very productive to fix those eslint issues rn.

**Comment by anuraghazra** [comment]
@kamleshchandnani this can be merged.

**Comment by chaitanyadeorukhkar** [comment]
> > everything is accessible in it's current state or are we missing any configuration?
> 
> If there is no aria. there's no errors :p
> 
> > There was errors but in blade-old I've ignored those, not very productive to fix those eslint issues rn.

So when does it fail? @anuraghazra

**Comment by anuraghazra** [comment]
> So when does it fail? @anuraghazra

Tests just fail. All expect calls fails.

File: .eslintrc.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+    'plugin:jsx-a11y/recommended',
+    'plugin:react-native-a11y/all',
+  ],
   rules: {
     'no-negated-condition': 'off',
     'max-lines-per-function': 'off',
     'max-lines': 'off',
     'no-console': 'off',
     'import/no-cycle': 'error',
+    'react-native-a11y/has-accessibility-hint': 'off',
```
[the remark this answers is not in the document]
Comment (anuraghazra) [reply .eslintrc.js:15]: Accessibility hint is not always needed, it's only needed if the accessibilityLabel is not providing enough information. It doesn't make sense to enforce this rule.

**Comment by anuraghazra** [comment]
> > > everything is accessible in it's current state or are we missing any configuration?
> > 
> > 
> > If there is no aria. there's no errors :p
> > > There was errors but in blade-old I've ignored those, not very productive to fix those eslint issues rn.
> 
> So when does it fail? @anuraghazra

If we have anything specific which might fail.. Currently we don't even have enough complex nested components that it will fail. If any of the rules mentioned in those two plugins fails it will yell at us.

**Review by kamleshchandnani — APPROVED** [review]

**Pull request merged**